src/goodbase/goodbase_cli.py

```diff
@@ -544,9 +553,19 @@ def dependencies(binder: inject.Binder) -> None:
         revision = orchestrator.checkout_good_base(evg_project, criteria)
 
         if revision:
-            click.echo(click.style(f"Found revision: {revision.revision}", fg="green"))
+            revision_dict = {}
+            revision_dict["mongo"] = revision.revision
```

Comment:
I have been trying to keep this tool agnostic about the repo it is running in. So it would be nice to not need to hard-code mongo here as that would not work for any other evergreen project.

Reply:
That makes sense. We can hardcode it to "stable_revision" instead, or use the repo name (worked out by `basename $(git remote get-url origin)`). Let me know if you have a preference.

Reply:
Addressed in 650b276